src/features/hotspots/root/HotspotsView.tsx

```diff
@@ -473,81 +411,45 @@ const HotspotsView = ({
           showNearbyHotspots
         />
         <HotspotMapButtons
-          animatedPosition={animatedDetailsPosition}
+          animatedPosition={animatedIndex}
```

Comment:
The animation for the witness button flickers heavily when you go from the full screen map to selecting a hotspot now